docs/rabbitmq-server.service.example

```diff
@@ -0,0 +1,17 @@
+# systemd unit example
+[Unit]
+Description=RabbitMQ broker
+After=syslog.target network.target
+
+[Service]
+Type=notify
+User=rabbitmq
+Group=rabbitmq
+NotifyAccess=all
+TimeoutStartSec=3600
```

Comment:
Recently I'm very suspicious of `infinity` timeouts. I don't remember who said something like: 
> When I see a timeouf of `inifinity`, I suggest to replace it with some insane value like 30 years. And if the reply to this is that 30 years doesn't make sense, I reply that infinity also makes no sense.

With `infinity` timeout some problem with startup could go unnoticed for a long period of time. The question is what is the sane timeout value - in my opininon it's no more than several hours.